Documentation/Getting-Started/resource-cleanup.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Ceph Cleanup
```

Comment:
Instead of a new doc, how about a new section in ceph-teardown.md?

Reply:
moved it to `ceph-teardown.md` doc.